scionlab/hostfiles/scionlab-config

```diff
@@ -298,20 +298,26 @@ def install_scionlab_service(tmpdir):
     scionlab_services = os.path.join(tmpdir, 'scionlab-services.txt')
     if not os.path.exists(scionlab_services):
         _error_exit("Could not find the scionlab-services.txt file in %s", tmpdir)
+    with open(scionlab_services, 'r') as f:
+        services = [service.strip() for service in f.readlines()]
+
+    # 1. Disable services no longer in the service list
+    res = subprocess.run(['systemctl', 'show', 'scion-*@*.service', '--property=Id'], stdout=subprocess.PIPE)
+    previous_services = [service.split("=")[-1].strip() for service in res.stdout.decode('utf-8').split("\n\n")]
+    to_disable = [service for service in previous_services if service not in services]
+    for srv in to_disable:
+        subprocess.run(['systemctl', 'disable', '--now', srv], check=True)
 
-    # 1. remove existing links to scionlab.target
+    # 2. remove existing links to scionlab.target
```

Comment:
the symlinks are already removed with the `systemctl disable` call.

Reply:
Done